src/services/movieService.ts

```diff
@@ -0,0 +1,48 @@
+import axios from "axios";
+import Movie from "../models/Movie";
+
+export default class MovieService {
+  baseURL = "http://localhost:8000";
```

Comment:
ezt a baseURL-t tedd ki az env-file ba, mert ez a változó helyi változó lesz, ha kikerül szerverre, ott majd nem a localhost:8000-et fogja meghívni. hanem egy éles szerver url-jét.

 Mivel a projektünk Vite-tel van kialakítva, akkor az env változókat a VITE_ prefixszel kell definiálni, és használni a kódban. Így az env-be így vedd fel: VITE_BASE_URL=https://localhost:8000

Itt pedig `baseURL = "http://localhost:8000";` helyett ez legyen `baseUrl = process.env.VITE_BASE_URL;`